catalog/input.yaml

```diff
@@ -0,0 +1,2 @@
+feedstocks:
+  - https://github.com/leap-stc/proto_feedstock/blob/main/feedstock/catalog.yaml
```

Comment:
@jbusecke, how does this look for feedstock registration?

Reply:
I like that. Would one be able to specify a PR branch (just out of curiosity)...

Reply:
yes. this flexibility makes it easier to work with since you can just name your branch whatever you want, and we don't have to make any assumption